src/uct/api/uct.h

```diff
@@ -1267,19 +1267,20 @@ typedef struct uct_md_mem_attr {
  * @brief Query attributes of a given pointer
  *
  * Return attributes such as memory type, and system device for the
- * given pointer of specific length.
+ * given pointer of specific length depending on fields requested in mem_attr.
  *
- * @param [in]     md          Memory domain to run the query on. This function
- *                             returns an error if the md does not recognize the
- *                             pointer.
+ * @param [in]     md          Memory domain to run the query on.
  * @param [in]     address     The address of the pointer. Must be non-NULL
  *                             else UCS_ERR_INVALID_PARAM error is returned.
  * @param [in]     length      Length of the memory region to examine.
  *                             Must be nonzero else UCS_ERR_INVALID_PARAM error
  *                             is returned.
- * @param [out]    mem_attr    If successful, filled with ptr attributes.
+ * @param [in/out] mem_attr    MD attribute param specifying fields of interest.
```

Comment:
maybe it should be `[inout]` like we have in `uct_worker_progress_register_safe()` for the `id_p` parameter?
could pls check if it fixes the doxygen? thank you!

Reply:
Thanks for pointing out @dmitrygx I've changed to `[in,out]` based on doxygen document check. Hopefully that fixes things and the notation doesn't change with doxygen version.

Reply:
@Akshay-Venkatesh 👍